detection_rules/etc/version.lock.json

```diff
@@ -11362,25 +11383,25 @@
       }
     },
     "rule_name": "Windows Firewall Disabled via PowerShell",
-    "sha256": "b832b358dba011c1161ed80e69f3b9fb97555b3257860df5116508c7b979a0dc",
+    "sha256": "042b248eef07037737bc0d472138c9557704ca65a5f29f0f3f50edc87d2a5e17",
     "type": "eql",
-    "version": 102
+    "version": 103
   },
   "f675872f-6d85-40a3-b502-c0d2ef101e92": {
     "min_stack_version": "8.3",
     "previous": {
       "7.16": {
         "max_allowable_version": 99,
         "rule_name": "Delete Volume USN Journal with Fsutil",
-        "sha256": "b8ea24d38fe9b6522e77bcc999fefa26a1ee87060cdb4e7d4d373dd994742272",
+        "sha256": "55e9fa64400f766306c6c956c730b863e3abb105aad98a773032dfd336d0ad27",
```

Comment:
Forked rule changed when all rules are min-stacked at 8.3 and thus changes would not backport to this branch.